src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx

```diff
@@ -10,18 +10,42 @@ import contextMapThreeEntities from 'images/context-map-three-entities.png'
 
 import contextMapEntityDetail from 'images/context-map-entity-detail.png'
 
-Some New Relic features, like distributed tracing and incident intelligence, display topological maps that show relationships between your New Relic-monitored entities and other external hosts and services. These maps can help you diagnose and correct issues.
+import omnimapServiceMaps from 'images/omnimapServiceMaps.png'
+
+import omnimapAutomapTooltip from 'images/omnimapAutomapTooltip.png'
+
+import omnimapServiceMapsTopNav from 'images/omnimapServiceMapsTopNav.png'
+
+import omnimapServiceMapsRightPane from 'images/omnimapServiceMapsRightPane.gif'
+
+New Relic can take data from instrumented entities and create topological maps that display the relationships between them. These entity-scoped maps use a source entity as an entry point into the rest of your system, letting you understand how your hosts, services, and applications relate and affect each other.
+
+If you want to learn more about Workloads maps, then check out [Intro to Workloads](/docs/new-relic-solutions/new-relic-one/introduction-new-relic-one). If you're looking to learn about automap, then read [Automap for troubleshooting entities](/docs/new-relic-solutions/new-relic-one/ui-data/automaps).
+
+## A consistent maps experience [#omni]
+
+With our omnimap-powered maps experience, we're able to unify disparate map capabilities into a single, consistent experience. Service maps, Workloads maps, and automap share key features and design, like [timewarp](/docs/new-relic-solutions/new-relic-one/ui-data/automaps/#timewarp-cursor), the **Health status** and **Entity types** toggle, and various UI elements. 
```

Comment:
this is first mention of 'omnimap' in this doc and I'm not clear what it's referring to. I get that it's 'unifying disparate map capabilities' into a single experience, but I'm not sure what that means from a customer perspective. Is it just a type of map? what does us being able to unify different maps have to do with that single map type? are there different maps under the hood that we're exposing as a single type of map and ,if so, would a customer care about that? Just a bit unclear what the 'omnimap' really is in practical sense.

Reply:
👍 to @zuluecho9 's point.

Perhaps something along the lines of "With our newest mapping engine, ..."